apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph-start-node/ba-decision-graph-start-node.html

```diff
@@ -0,0 +1,17 @@
+<div class="ba-uxd-startnode-grid">
+  <div *ngFor="let startnode of decisionGraphStartnodes, let i = index">
+    <button
+      class="ba-uxd-startnode"
+      [ngClass]="{
+        'dt-uxd-startnode-visited': isSelectedStartnode(i)
+      }"
+      id="{{i}}"
+      (click)="selectStartnode(startnode)"
+    >
+      <div
```

Comment:
technically, buttons are inline elements and should not have nested block level elements. 
Is there are lot of complex markup content in the `getSanitizedNodeText(startnode.text)`? If not I would argue you can put the content directly into the button html and remove the div.